source/index.txt

```diff
@@ -43,12 +43,6 @@ Get Started
 Learn how to install the driver, establish a connection to MongoDB, and begin
 working with data in the :ref:`scala-get-started` tutorial.
 
-Connect to MongoDB
-------------------
-
-Learn how to create and configure a connection to a MongoDB deployment
-in the :ref:`scala-connect` section.
```

Comment:
I: I disagree that we should remove the link here, as there is a whole connection drawer in the TOC. 

S: Instead of removing this link, can you add basic content to the landing page that just lists the drawer pages? See https://github.com/mongodb/docs-kotlin/blob/master/source/fundamentals/connection.txt as an example